.github/workflows/ci.yml

```diff
@@ -1,21 +1,19 @@
 name: CI
 
 # Runs build and test on:
-#   every push to main that has a change in a file not in the docs folder
-#   every pull request with main branch as the base that has a change in a file not in the docs folder
+#   every push to main that has a change in a file
+#   every pull request with main branch as the base that has a change in a file
 on:
   push:
     branches:
       - main
     paths:
```

Comment:
Why don't we just remove the entire `paths` blocks?

Reply:
Yeah that seems like a good idea!